pkg/Makefile.am

```diff
@@ -4,12 +4,21 @@ pkg_TESTS = \
 
 TESTS += $(pkg_TESTS)
 
+metainfo_in = \
+	pkg/sosreport/org.cockpit-project.cockpit-sosreport.metainfo.xml \
+	pkg/kdump/org.cockpit-project.cockpit-kdump.metainfo.xml \
+	pkg/selinux/org.cockpit-project.cockpit-selinux.metainfo.xml \
+	pkg/storaged/org.cockpit-project.cockpit-storaged.metainfo.xml \
+	$(NULL)
+EXTRA_DIST += $(metainfo_in)
+
+dist/%.metainfo.xml: pkg/%.metainfo.xml
+	$(AM_V_GEN) mkdir -p $(dir $@) && msgfmt --xml -d $(top_srcdir)/po --template $< --output $@
+
 metainfodir = ${datarootdir}/metainfo
-metainfo_DATA = pkg/sosreport/org.cockpit-project.cockpit-sosreport.metainfo.xml \
-		pkg/kdump/org.cockpit-project.cockpit-kdump.metainfo.xml \
-		pkg/selinux/org.cockpit-project.cockpit-selinux.metainfo.xml \
-		pkg/storaged/org.cockpit-project.cockpit-storaged.metainfo.xml \
-		$(NULL)
+metainfo_DATA = $(patsubst pkg/%,dist/%,$(metainfo_in))
```

Comment:
As a general rule, I prefer to write the names of the installed files in the longer list, and derive the `.in` files from that.  `src/systemd/Makefile.am` is a good example of that.

Let's just do the usual `.xml.in` → `.xml` in place thing, no?

Reply:
Done. Thanks for the guidance! I also changed it so that the merged files are not disted, but built, same like our .appdata.xml.in. I *think* with that we can also re-drop the new packit srpm dependency, tests will tell.